packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -1246,7 +1359,7 @@ async function handler(req: NextApiRequest & { userId?: number | undefined }) {
   try {
     await scheduleWorkflowReminders(
       eventType.workflows,
-      reqBody.smsReminderNumber as string | null,
+      smsReminderNumber as string | null,
```

Comment:
Turns out smsReminderNumber is not expected to be undefined by `scheduleWorkflowReminders` but it can be undefined. Handled that and removed this type assertion